### PR TITLE
OSDOCS-1648: Adding API dashboard to release notes

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1648

Preview: https://deploy-preview-29025--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-monitoring-api-performance-dashboard

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -102,6 +102,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+The API performance dashboard is now available from the web console. This dashboard can be used to help troubleshoot performance issues with the Kubernetes API server or the OpenShift API server. You can access the API performance dashboard from the web console by navigating to *Monitoring* -> *Dashboards* and selecting the *API Performance* dashboard.
+
+This dashboard provides API server metrics, such as:
+
+* Request duration
+* Request rate
+* Request latency
+* Request termination
+* Requests in flight
+* Long running requests
```
Comment (bobfuru) [inline release_notes/ocp-4-7-release-notes.adoc:117]: s/Long running requests/Long-running requests
Not sure if this is matching the dashboard GUI? If so, then disregard this suggestion.

**Review by bobfuru — COMMENTED** [review]
One possible edit but otherwise LGTM!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-7-release-notes.adoc:117]: Nope, not trying to match GUI labels, thanks!

**Comment by bergerhoffer** [comment]
@vikram-redhat FYI I'm seeing two different preview bots commenting on this PR (with two different links). Not sure if the old one shouldn't be enabled still?

**Comment by bergerhoffer** [comment]
@xingxingxia Can you please review these release notes for the new API performance dashboard? Thanks!

https://deploy-preview-29025--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-monitoring-api-performance-dashboard

**Comment by vikram-redhat** [comment]
> @vikram-redhat FYI I'm seeing two different preview bots commenting on this PR (with two different links). Not sure if the old one shouldn't be enabled still?

Yes, I haven't removed the old preview from enterprise-4.7 yet, only master. Will do this shortly. You can delete the comment for the old preview.

**Comment by xingxingxia** [comment]
LGTM, thanks

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -102,6 +102,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+[id="ocp-4-7-monitoring-api-performance-dashboard"]
+==== New API performance monitoring dashboard
+
+The API performance dashboard is now available from the web console. This dashboard can be used to help troubleshoot performance issues with the Kubernetes API server or the OpenShift API server. You can access the API performance dashboard from the web console by navigating to *Monitoring* -> *Dashboards* and selecting the *API Performance* dashboard.
+
+This dashboard provides API server metrics, such as:
+
+* Request duration
+* Request rate
+* Request latency
```
Comment (tkashem) [inline release_notes/ocp-4-7-release-notes.adoc:114]: `duration` and `latency` both refer to the same metric.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -102,6 +102,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+* Request termination
+* Requests in flight
+* Long-running requests
+* Response status code
+* Response size
+* Priority and fairness
+
```
Comment (tkashem) [inline release_notes/ocp-4-7-release-notes.adoc:121]: we also have:
- etcd (request duration and object count)
- request aborted

**Pull request merged**